### ISSUE TITLE
Dice: Removing All Faces on 3-D Dice Crashes Client

### ISSUE THREAD

**Issue by 96LawDawg** [ArnoldSmith86/virtualtabletop#1677]
Well, it didn't take too long to find the first dice problem.

Add a 3-D dice.  Delete all the face information (using the JSON Editor or routines) leaving just [ ].  Crashes client.  Console says: "Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'."

**Comment by RaphaelAlvez** [comment]
how are you removing all faces? There is a default so I would expect it to solve the problem

**Comment by 96LawDawg** [comment]
First, I did it in the JSON Editor.  I want a d6.  I wanted to type some names in there so I erased the default 1-6.  Got the error.  That shouldn't happen.  Needs to be fixed.

Then I also tried it programmatically.  I have an array of names that starts out blank at the beginning of the game then gets filled with player names as they take their seats.  I have the dice get the current array of players and make the face property be that array.  If it is empty, the room breaks.
